pkg/controller/storagecluster/stork.go

```diff
@@ -823,6 +820,15 @@ func getStorkServiceLabels() map[string]string {
 	}
 }
 
+func imagePullPolicy(cluster *corev1alpha1.StorageCluster) v1.PullPolicy {
```

Comment:
why this special handling for stork ?

Reply:
As discussed, we have the same method for all portworx components. So added the same for Stork.